docs/lakebridge/docs/installation.mdx

```diff
@@ -111,14 +122,38 @@ databricks configure --host <host> --profile <profile_name>
 
 ----
 
-## Install Transpile
+## Install Lakebridge
 
 ### Installation
 
 Upon completing the environment setup, install Lakebridge by executing the following command:
 ```bash
 databricks labs install lakebridge
 ```
+<img src={useBaseUrl('img/Install_Lakebridge.gif')} alt="lakebridge-install" />
+
+### Verify Installation
+Verify the successful installation by executing the provided command;
+confirmation of a successful installation is indicated when the displayed output aligns with the example below:
+```bash
+ databricks labs lakebridge transpile --help
```

Comment:
This works, but I think this would be better:
```suggestion
 databricks labs lakebridge --help
```
(Luckily this already matches the screencap below.)